app/views/account/users/_user.slim

```diff
@@ -1,4 +1,13 @@
 li
   = link_to user.email, user.email
   span
-    = user.name
\ No newline at end of file
+    = " | Name: "
+  span
+    = user.name
+  - if user.role == 'reporter'
+    span
+      = " | "
+    - if user.banned?
```

Comment:
so admin won't be able to edit user? 
maybe just add a form and checkbox on Edit User page and edit this checkbox?